FluidNC/src/Motors/Dynamixel2.cpp

```diff
@@ -70,11 +74,11 @@ namespace MotorDrivers {
         LED_on(true);
         vTaskDelay(100);
         LED_on(false);
-
-        startUpdateTask(_timer_ms);
     }
 
-    void Dynamixel2::config_message() { log_info("    " << name() << " id::" << _id << " Count(" << _countMin << "," << _countMax << ")"); }
+    void Dynamixel2::config_message() {
+        log_info("    " << name() << " id::" << _id << " Count(" << _countMin << "," << _countMax << ")");
```

Comment:
Is id:: a typo?  Should it be just id:  ?

Reply:
yes